pallets/reversible-transfers/src/tests/test_reversible_transfers.rs

```diff
@@ -1107,11 +1104,10 @@ fn full_flow_cancel_prevents_execution_with_timestamp_delay() {
 		let interceptor_balance = Balances::free_balance(interceptor_255());
 		assert_eq!(interceptor_balance, amount); // interceptor started with 0, now has the cancelled amount
 
-		let expected_event_pattern = |e: &RuntimeEvent| match e {
-			RuntimeEvent::ReversibleTransfers(Event::TransactionExecuted {
-				tx_id: tid, ..
-			}) if *tid == tx_id => true,
-			_ => false,
+		let expected_event_pattern = |e: &RuntimeEvent| {
+			matches!(e, RuntimeEvent::ReversibleTransfers(Event::TransactionExecuted {
+			tx_id: tid, ..
+		}) if *tid == tx_id)
 		};
```

Comment:
its not clear to me why this returns the same as it did before. It probably does but I am not a fan of this totally needless change.